### PR TITLE
Delegate more functions to ACL

### LINKED ISSUE #301: Delegate more functions to ACL entries
https://github.com/gimite/google-drive-ruby/blob/master/lib/google_drive/acl.rb#L30

Please delegate more functions such as `any?`, `none?`, etc.

### PR DESCRIPTION
**Pull request by StephanMeijer**
Closes #301

### CODE REVIEW

File: lib/google_drive/acl.rb [hunk 1]
```diff
@@ -27,7 +27,9 @@ def initialize(session, file)
         api_permissions.permissions.map { |perm| AclEntry.new(perm, self) }
     end
 
-    def_delegators(:@entries, :size, :[], :each)
+    def_delegators(:@entries, :size, :[], :each, :map, :any?, :sample, :select,
```
Comment (gimite) [inline lib/google_drive/acl.rb]: Thanks! But can you let this class include Enumerable instead? It would add most of the methods here and even more.

**Comment by StephanMeijer** [comment]
Done @gimite

File: lib/google_drive/acl.rb [hunk 2]
```diff
@@ -14,6 +14,7 @@ module GoogleDrive
   # This code is based on https://github.com/guyboertje/gdata-spreadsheet-ruby .
   class Acl
     include(Util)
+    include(Enumerable)
```
Comment (gimite) [inline lib/google_drive/acl.rb]: Well, you still need to keep original delegates (`size`, `[]`, `each`). `size` and `[]` are not part of Enumerable. `each` is needed to make Enumerable work.

Comment (StephanMeijer) [reply lib/google_drive/acl.rb]: Ah, I already wondered how it would work.

Comment (StephanMeijer) [reply lib/google_drive/acl.rb]: Done!

**Pull request merged**